### ISSUE TITLE
Crash while importing fbx file in 4.4-dev3

### ISSUE THREAD

**Issue by jamie-pate** [godotengine/godot#98902]
### Tested versions

- reproducible in 4.4-dev3
- Not reproducible in 4.4-dev2

### System information

Godot v4.4.dev3 - Ubuntu 22.04.5 LTS 22.04 - X11 - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 3050 Laptop GPU - 12th Gen Intel(R) Core(TM) i7-12700H (14 Threads)

### Issue description

Editor output
```
EditorFileSystem: Importing file: res://makes-godot-4-4-dev3-crash.fbx
FBX path: res://makes-godot-4-4-dev3-crash.fbx
EditorFileSystem: Importing file: res://makes-godot-4-4-dev3-crash_0.png
EditorFileSystem: "res://makes-godot-4-4-dev3-crash_0.png" import took 7 ms.
Loading resource: res://makes-godot-4-4-dev3-crash_0.png
FBX: Total images: 1
Total materials: 1
FBX: Total skins: 1
FBX: Parsing mesh: 0
FBX: Total meshes: 1
FBX: Total lights: 0
FBX: Total cameras: 0
FBX: Total animations '1'.
FBX: Creating mesh for: unamed
Saving scene to: res://.godot/imported/makes-godot-4-4-dev3-crash.fbx-002f337880c2de989058e81dc3a2c321.scn
EditorFileSystem: "res://makes-godot-4-4-dev3-crash.fbx" import took 204 ms.
Using present mode: Enabled
Using present mode: Enabled
res://makes-godot-4-4-dev3-crash_0.png: Texture detected as used in 3D. Enabling mipmap generation and setting the texture compression mode to VRAM Compressed (S3TC/ETC/BPTC).
Using present mode: Enabled
Using present mode: Disabled
EditorFileSystem: Importing file: res://makes-godot-4-4-dev3-crash_0.png
etcpak: Encoding took 0 ms.
EditorFileSystem: "res://makes-godot-4-4-dev3-crash_0.png" import took 9 ms.
Using present mode: Enabled
Generated 'res://makes-godot-4-4-dev3-crash.fbx' preview in 209 usec
Loading resource: res://makes-godot-4-4-dev3-crash_0.png
bcdec: Decompression of a 256x1 DXT5 RGBA8 image with 0 mipmaps took 0 ms.
double free or corruption (!prev)
Aborted

```

Stack Trace:
```
Memory::free_static(void * p_ptr, bool p_pad_align) (/home/jpate/build/src/godot/core/os/memory.cpp:201)
CowData<unsigned char>::_unref(CowData<unsigned char> * const this) (/home/jpate/build/src/godot/core/templates/cowdata.h:270)
CowData<unsigned char>::~CowData(CowData<unsigned char> * const this) (/home/jpate/build/src/godot/core/templates/cowdata.h:478)
Vector<unsigned char>::~Vector(Vector<unsigned char> * const this) (/home/jpate/build/src/godot/core/templates/vector.h:291)
Image::~Image(Image * const this) (/home/jpate/build/src/godot/core/io/image.h:343)
memdelete<Image>(Image * p_class) (/home/jpate/build/src/godot/core/os/memory.h:140)
Ref<Image>::unref(Ref<Image> * const this) (/home/jpate/build/src/godot/core/object/ref_counted.h:215)
Ref<Image>::~Ref(Ref<Image> * const this) (/home/jpate/build/src/godot/core/object/ref_counted.h:228)
FBXDocument::_parse_materials(FBXDocument * const this, Ref<FBXState> p_state) (/home/jpate/build/src/godot/modules/fbx/fbx_document.cpp:1236)
FBXDocument::_parse_fbx_state(FBXDocument * const this, Ref<FBXState> p_state, const String & p_search_path) (/home/jpate/build/src/godot/modules/fbx/fbx_document.cpp:2181)
FBXDocument::_parse(FBXDocument * const this, Ref<FBXState> p_state, String p_path, Ref<FileAccess> p_file) (/home/jpate/build/src/godot/modules/fbx/fbx_document.cpp:2106)
FBXDocument::append_from_file(FBXDocument * const this, String p_path, Ref<GLTFState> p_state, uint32_t p_flags, String p_base_path) (/home/jpate/build/src/godot/core/object/ref_counted.h:209)
EditorSceneFormatImporterUFBX::import_scene(EditorSceneFormatImporterUFBX * const this, const String & p_path, uint32_t p_flags, const HashMap<StringName, Variant, HashMapHasherDefault, HashMapComparatorDefault<StringName>, DefaultTypedAllocator<HashMapElement<StringName, Variant> > > & p_options, List<String, DefaultAllocator> * r_missing_deps, Error * r_err) (/home/jpate/build/src/godot/modules/fbx/editor/editor_scene_importer_ufbx.cpp:81)
ResourceImporterScene::import(ResourceImporterScene * const this, const String & p_source_file, const String & p_save_path, const HashMap<StringName, Variant, HashMapHasherDefault, HashMapComparatorDefault<StringName>, DefaultTypedAllocator<HashMapElement<StringName, Variant> > > & p_options, List<String, DefaultAllocator> * r_platform_variants, List<String, DefaultAllocator> * r_gen_files, Variant * r_metadata) (/home/jpate/build/src/godot/editor/import/3d/resource_importer_scene.cpp:2974)
EditorFileSystem::_reimport_file(EditorFileSystem * const this, const String & p_file, const HashMap<StringName, Variant, HashMapHasherDefault, HashMapComparatorDefault<StringName>, DefaultTypedAllocator<HashMapElement<StringName, Variant> > > & p_custom_options, const String & p_custom_importer, Variant * p_generator_parameters, bool p_update_file_system) (/home/jpate/build/src/godot/editor/editor_file_system.cpp:2728)
EditorFileSystem::reimport_files(EditorFileSystem * const this, const Vector<String> & p_files) (/home/jpate/build/src/godot/editor/editor_file_system.cpp:3035)
EditorFileSystem::_update_scan_actions(EditorFileSystem * const this) (/home/jpate/build/src/godot/editor/editor_file_system.cpp:937)
EditorFileSystem::_notification(EditorFileSystem * const this, int p_what) (/home/jpate/build/src/godot/editor/editor_file_system.cpp:1667)
EditorFileSystem::_notificationv(EditorFileSystem * const this, int p_notification, bool p_reversed) (/home/jpate/build/src/godot/editor/editor_file_system.h:145)
Object::notification(Object * const this, int p_notification, bool p_reversed) (/home/jpate/build/src/godot/core/object/object.cpp:875)
SceneTree::_process_group(SceneTree * const this, SceneTree::ProcessGroup * p_group, bool p_physics) (/home/jpate/build/src/godot/scene/main/scene_tree.cpp:1020)
SceneTree::_process(SceneTree * const this, bool p_physics) (/home/jpate/build/src/godot/scene/main/scene_tree.cpp:1097)
SceneTree::process(SceneTree * const this, double p_time) (/home/jpate/build/src/godot/scene/main/scene_tree.cpp:580)
Main::iteration() (/home/jpate/build/src/godot/main/main.cpp:4380)
OS_LinuxBSD::run(OS_LinuxBSD * const this) (/home/jpate/build/src/godot/platform/linuxbsd/os_linuxbsd.cpp:962)
main(int argc, char ** argv) (/home/jpate/build/src/godot/platform/linuxbsd/godot_linuxbsd.cpp:85)
```

### Steps to reproduce

Remove the `.godot` folder and then try to open the MRP project in the editor.

### Minimal reproduction project (MRP)

[fbx-import-crash1.zip](https://github.com/user-attachments/files/17652126/fbx-import-crash1.zip)

**Comment by jamie-pate** [comment]
I get this while trying to decode the following test image directly with bcdec:
![256x1](https://github.com/user-attachments/assets/9ee11db8-61bd-4c9a-abd1-98ac765b143c)
[256x1_test_image.zip](https://github.com/user-attachments/files/17653406/256x1_test_image.zip)

```
./test ./test_images/256x1_bc3.dds 
Successfully loaded ./test_images/256x1_bc3.dds
 w = 256, h = 1, format = BC3
Writing output to ./test_images/256x1_bc3.tga
malloc(): invalid size (unsorted)
Aborted
```

**Comment by fire** [comment]
Do you think we can fix 4.4. release? Don’t want to revert work.

**Comment by iOrange** [comment]
Looks like this was addressed here - https://github.com/godotengine/godot/commit/a60195e23d99a1b1fda61bcee023b56160fd4096

**Comment by akien-mga** [comment]
Fixed by #97873.
